src/main/java/com/limechain/runtime/Runtime.java

```diff
@@ -32,4 +31,8 @@ public Object call(String functionName) {
         //TODO Call adequate params
```

Comment:
This TODO isn't clear. `adequate` is has unclear meaning. Please refactor it.

Reply:
I'm not sure to what to refactor it. It was added from Bobi, not from me